Specifications/Events/for_EventStore/when_committing_a_public_event_with_an_event_type.cs

```diff
@@ -0,0 +1,34 @@
+// Copyright (c) Dolittle. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Threading;
+using Dolittle.SDK.Events.for_EventConverter.given;
+using Dolittle.SDK.Protobuf;
+using Machine.Specifications;
+using Moq;
+using It = Machine.Specifications.It;
+
+namespace Dolittle.SDK.Events.for_EventStore
+{
+    public class when_committing_a_public_event_with_an_event_type : given.an_event_store_and_an_execution_context
+    {
+        static an_event content;
+        static EventSourceId event_source;
+        static EventType event_type;
+
+        Establish context = () =>
+        {
+            content = new an_event("goodbye public world", 12345, true);
+            event_source = new EventSourceId("be274a66-e442-455a-b5d9-81d058fc179e");
+            event_type = new EventType("31819551-844f-4a6f-935c-917a126531f6");
+        };
+
+        Because of = async () => await event_store.CommitPublic(content, event_source, event_type);
```

Comment:
As far as I know - `async` is not supported by Machine.Specifications. So we need to wait with a blocking call.

I think we also want to make sure that the returned result from the `CommitPublic` is the result from the `EventConverter` - right?

Reply:
`async` support seems to be a [_"maybe"_](https://github.com/machine/machine.specifications/issues/293#issuecomment-655909400) so yeah, better to just keep it blocking.